src/Widgets/LeeanPreview.php

```diff
@@ -0,0 +1,167 @@
+<?php namespace Leean\Widgets\Widgets;
+
+use Leean\Widgets\Models\AbstractWidget;
+
+/**
+ * Class LeeanPreview.
+ *
+ * @package Leean\Widgets\Widgets
+ */
+class LeeanPreview extends AbstractWidget
+{
+	/**
+	 * LeeanPreview constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'Leean Preview', 'Display previews (header/description/link)' );
+	}
+
+	/**
+	 * POst registration.
+	 */
+	public static function post_registration() {
+		if ( function_exists( 'acf_add_local_field_group' ) ) :
+
+			acf_add_local_field_group( array(
+				'key' => 'group_56ef50a64f75f',
+				'title' => 'LeeanPreview',
+				'fields' => array(
+					array(
+						'key' => 'field_56ef50b46191d',
```

Comment:
LeeanPreview is a pre-defined widget which is included with this module. So these acf fields are only used for this widget.